addusers.py

```diff
@@ -217,16 +217,32 @@ def exists_in_keystone(check_thing, keystone_things):
         return None
 
 
-def parse_rows(rows):
+def parse_rows(rows, select_user=None):
     """Parse spreadsheet user/project data into User and Project classes
     
     Expects 'rows' to include all rows, with row 0 being the header row
     Returns a dictionary of projects keyed by project name, and a list
     of rows that were not blank but failed to parse correctly.
+
+    Select_user allows caller to handle requests from one user only.
     """
+    USER_COLUMN = 3
```

Comment:
Maybe have a note/reference here? Since this USER_COLUMN would change if we change the order of the questions in the form, right?